packages/migrate/src/__tests__/fixtures/schema-only-data-proxy/with-directUrl-env.prisma

```diff
@@ -0,0 +1,11 @@
+generator client {
+    provider = "prisma-client-js"
+}
+
+datasource db {
+    provider  = "postgresql"
+    url       = "this should just fail"
```

Comment:
This invalid value is not how a user would use the Data Proxy, they would have `prisma://aws-us-east-1.prisma-data.com/?api_key=MY_API_KEY` in there which currently makes validation fail :(